mobile_verifier/src/boosted_hexes.rs

```diff
@@ -0,0 +1,41 @@
+use chrono::{DateTime, Utc};
+use futures::StreamExt;
+use mobile_config::{
+    boosted_hex_info::BoostedHexInfo,
+    client::{hex_boosting_client::HexBoostingInfoResolver, ClientError},
+};
+use std::collections::HashMap;
+
+#[derive(Debug, Clone, Default)]
+pub struct BoostedHexes {
+    pub hexes: HashMap<u64, BoostedHexInfo>,
+}
+
+#[derive(PartialEq, Debug, Clone)]
+pub struct BoostedHex {
+    pub location: u64,
+    pub multiplier: u32,
```

Comment:
shouldn't this be a Decimal?

Reply:
the onchain boosts values are u8's

Reply:
yeah, but you have to multiply this with a decimal somewhere, and that requires converting it to a decimal. So my thinking is that you should convert this to a decimal while it is being used by mobile verifier, and convert it back when you're writing to a protobuf

Reply:
i kept this as is, the input and outputs are all integers and I convert at the point of a singular need to decimal